src/Extensions/ExtHostClient.rei

```diff
@@ -55,11 +159,12 @@ let start:
   ) =>
   t;
 let activateByEvent: (string, t) => unit;
-let executeContributedCommand: (string, t) => unit;
+let executeContributedCommand:
+  (~arguments: list(Core.Json.t)=?, string, t) => unit;
```

Comment:
Added `~arguments`, since that's needed for some commands.

Reply:
Excellent!